src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -44,6 +46,18 @@ else if ( value instanceof Timestamp )
 			return "";
 	}
 
+	private String formatLong(Long value) {
+		final int second = (int)(value % 100); value /= 100;
+		final int minute = (int)(value % 100); value /= 100;
+		final int hour = (int)(value % 100); value /= 100;
+		final int day = (int)(value % 100); value /= 100;
+		final int month = (int)(value % 100);
+		final int year = (int)(value / 100);
```

Comment:
no, this is mysql itself.

On Mon, Mar 09, 2015 at 03:08:36PM -0700, Vanchi Koduvayur wrote:
> > @@ -44,6 +46,18 @@ else if ( value instanceof Timestamp )
> >  			return "";
> >  	}
> >
> > +	private String formatLong(Long value) {
> > +		final int second = (int)(value % 100); value /= 100;
> > +		final int minute = (int)(value % 100); value /= 100;
> > +		final int hour = (int)(value % 100); value /= 100;
> > +		final int day = (int)(value % 100); value /= 100;
> > +		final int month = (int)(value % 100);
> > +		final int year = (int)(value / 100);
>
> Hm. Is the open replicator doing this encoding underneath? "2014-01-02 12:40:56" => 20140102124056
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/zendesk/maxwell/pull/15/files#r26081736
>